manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -907,32 +939,47 @@ Ext.extend(MODx.grid.Grid,Ext.grid.EditorGridPanel,{
     /**
      * @property {Function} clearGridFilters - Clears all grid filters and sets them to their default value
      *
-     * @param {String} itemIds - A comma-separated list of the Ext component ids to be cleared
+     * @param {String|Array} items - A comma-separated list (or array) of items to be cleared. An optional default value
+     * may also be specified. The expected format for each item in the list is:
+     * 'filter-category', where 'filter-category' matches the Ext component's itemId and 'category' is the record index to filter on OR
+     * 'filter-category:3', where '3' is the filter's default value to be applied (instead of setting to an empty value)
+     *
      */
-    ,clearGridFilters: function(itemIds) {
+    ,clearGridFilters: function(items) {
         const store = this.getStore(),
-              bottomToolbar = this.getBottomToolbar()
+              bottomToolbar = this.getBottomToolbar(),
+              data = Array.isArray(items) ? items : items.split(',')
         ;
-        itemIds = Array.isArray(itemIds) ? itemIds : itemIds.split(',');
-        /*
-            Note that param below relies on the following naming convention being followed for each filter's config:
-            itemId: 'filter-category', where 'category' is the record index to filter on
-        */
-        itemIds.forEach(itemId => {
-            const id = itemId.trim(),
-                  cmp = this.getFilterComponent(id)
+        data.forEach(item => {
+            itemData = item.replace(/\s+/g, '').split(':');
```

Comment:
Where is `itemData` variable declaration?

Reply:
Good catch - fixed that on my side.